defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# defaults file for sanity-check
+
+SANITY_CHECK_DISK_SPACE:
+    # Mount point
+  - mount: /
+    # Treshold in mb
+    treshold_mb: 5000
```

Comment:
This is doable. For me mb's are cleaner as if I can estimate how long it will take me to increase the drive size, and how much data is stored I can estimate safety marigin in mbs. But percents are fine too.